services/repology/repology-repositories.service.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { nonNegativeInteger } = require('../validators')
+const { BaseSvgScrapingService } = require('..')
+
+const schema = Joi.object({
+  message: nonNegativeInteger,
+}).required()
+
+module.exports = class RepologyRepositories extends BaseSvgScrapingService {
+  static get category() {
+    return 'platform-support'
+  }
+
+  static get route() {
+    return {
+      base: 'repology/repositories',
+      pattern: ':projectName',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Repology - Repositories',
+        namedParams: { projectName: 'starship' },
+        staticPreview: this.render({ repositoryCount: '18' }),
+        keywords: ['repositories'],
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return { label: 'in repositories' }
+  }
+
+  static render({ repositoryCount }) {
+    return {
+      message: repositoryCount,
+      color: 'blue',
```

Comment:
and if the color is going to be static, you can move this up to `defaultBadgeData`